scripts/jestSetup.js

```diff
@@ -15,3 +15,13 @@ global.IntersectionObserver = class {
   unobserve() {}
   disconnect() {}
 };
+
+const MockReact = jest.requireActual('react');
+const MockReactDOM = jest.requireActual('react-dom');
```

Comment:
nit - aren't these the actual modules? The variable names are wrong?

Reply:
jest will throw an error if you try to access variables outside the mock factory's scope unless you whitelist them by prefixing them with "mock". 🤷‍♀️
https://github.com/facebook/jest/blob/5fe8b0/packages/babel-plugin-jest-hoist/src/index.ts#L140-L152

Reply:
lovely